src/dso_api/dynamic_api/permissions.py

```diff
@@ -142,35 +149,33 @@ def _filters_ok(request, schema: DatasetTableSchema) -> bool:  # noqa: C901
         mandatory.update(*profile.mandatory_filtersets)
 
     for key, values in request.GET.lists():
-        if key in ("fields", "format", "page", "page_size", "sorteer") or key.startswith("_"):
+        if key in allowed:
             continue
 
         # Everything else is a filter.
         field_name, op = _parse_filter(key)
 
+        # OPTIONS requests should validate, but are not
```

Comment:
Seems like the sentence is not finished?

Reply:
This shouldn't have been here at all. OPTIONS is already checked at function entry time. Removed.